lib/NEW-STREAMER-SETUP-07-04-26/setup.sh

```
@@ -0,0 +1,444 @@
+echo "       Run:    pm2 status"
+echo "       Expect: streamer-$DROPSHOT_GROUND_ID shown as 'online'"
+echo "       Logs:   pm2 logs streamer-$DROPSHOT_GROUND_ID"
+echo ""
+echo "  5. TEST AUTO-START ON REBOOT"
+echo "       Reboot the machine, then confirm:"
+echo "       - pm2 status       → streamer is online"
+echo "       - tailscale status → connected"
+echo ""
+echo -e "${YELLOW}=============================================${NC}"
+echo ""
+echo "  1. BIOS AUTO POWER-ON (AC RECOVERY)
```

Shellcheck (0.11.0)

[warning] Did you forget to close this double quoted string?

(SC1078)